security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainResourceDefinition.java

```diff
@@ -63,9 +64,12 @@
  */
 class SecurityDomainResourceDefinition extends SimpleResourceDefinition {
 
+    static final String INFINISPAN_CACHE_TYPE = "infinispan";
+    static final RuntimeCapability<Void> INFINISPAN = RuntimeCapability.Builder.of("org.wildfly.security.domain.infinispan", true).build();
```

Comment:
@bstansberry #9742 works for me.  This issue isn't holding me up - the associated jira has no dependents.